src/pint/fitter.py

```diff
@@ -569,7 +569,7 @@ def get_correlation_matrix(self, with_phase=False, pretty_print=False, prec=3):
             )
             raise AttributeError
 
-    def ftest(self, parameter, component, remove=False, full_output=False):
+    def ftest(self, parameter, component, remove=False, full_output=False, NITS=1):
```

Comment:
This parameter is called `maxiter` in `fit_toas`; it would be nice if PINT were consistent with itself.

Reply:
Done!